app/models/test_track/ab_configuration.rb

```diff
@@ -9,7 +9,10 @@ def initialize(opts)
 
     @true_variant = true_variant.to_s if true_variant
 
-    raise ArgumentError, "unknown split: #{split_name}" if @split_registry && !split
+    if @split_registry && !split
+      raise ArgumentError, "unknown split: #{split_name}. " \
+        "#{'You may need to run rake test_track:schema:load' unless Rails.env.production?}"
```

Comment:
I think this should be specific to development - in my experience most environments other than development are long-lived staging-type envs.